scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+  Notes:
+- group: simulator
+  name: mode_mask
+  type: packed bitfield
+  units:
+  default value: 15 (decimal), 0xF (hexadecimal)
+  enumerated possible values:
+  Description: 'Determines the types of position outputs for the simulator.  '
+  Notes: |-
+    bit 0 (decimal value 1) turns on single point position PVT simulated outputs
+    bit 1 (decimal value 2) turns on the satellite tracking simulated outputs
+    bit 2 (decimal value 4) turns on Float IAR simulated RTK outputs
+    bit 3 (decimal value 8) turns on Fixed IAR simulated RTK outputs
+- group: solution
+  name: soln_freq
+  type: integer
+  units: hz
+  default value: '10'
+  enumerated possible values:
+  Description: The rate at which a solution is generated internally to the Piksi.
+  Notes:
+- group: solution
+  name: output_every_n_obs
+  type: integer
+  units:
+  default value: '2'
+  enumerated possible values:
+  Description: Integer divisor of solution frequency for which the observations will
+    be output.
+  Notes: For instance, if the solution frequency is 10 hz, and the "output every n
+    obs" parameter is 2, it means that the observation output will occur at a rate
+    of 5hz.  Since the observations are the information used by the Piksi receiving
+    corrections from the connected Piksi, this determines the rate of information
+    sharing for RTK solution output. This parameter is designed to tune the rate at
+    which correction information is passed from one Piksi to the other as to efficiently
+    use radio modem bandwidth and fit with user applications.
+- group: solution
+  name: dgnss_solution_mode
+  type: enum
+  units:
+  default value:
+  enumerated possible values: Low Latency, Time Matched
+  Description: Determines the type of RTK solution which will be output.
+  Notes: A low latency solution uses an internal model of anticipated satellite observations
+    to provide RTK output with minimal latency but slightly reduced accuracy.  This
+    mode should be used for most applications.  Where accuracy is desired with the
+    expectation of increased latency, the "Time matched" mode can be chosen.  This
+    means that the RTK output will require a corresponding set of correction observations
+    for each timestamp.
+- group: solution
+  name: dgnss_filter
+  type: enum
+  units:
+  default value: Fixed
+  enumerated possible values: Fixed,Float
+  Description: Determines the type of IAR the pikis will attempt to achieve.
+  Notes: If "fixed", the Piksi will output a fixed integer ambiguity estimate.  If
+    no fixed solution is available, it will rever to the float solution.  If "float",
+    the device will only output the float integer ambiguity estimate.
+- group: solution
+  name: known_baseline_n
+  type: double
+  units: meters (north)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
```

Comment:
Maybe explain this is the number of meters that the rover is north from the base station.

Also for E and D